src/app/core/services/signing/signing.service.ts

```diff
@@ -36,11 +42,110 @@ export class SigningService {
     });
   }
 
+  signUpdateEntry(didId: string, selectedManagementKey: ManagementKeyEntryModel, entry: UpdateEntryDocument, vaultPassword: string): Observable<SignatureResultModel> {
+    return defer(async () => {
+      try {
+        const vault = this.vaultService.getVault();
+        const decryptedVault = await encryptor.decrypt(vaultPassword, vault);
+        const managementKeys = decryptedVault[didId].managementKeys;
```

Comment:
Add TODO to check if it's possible that all management keys for a DID are removed.